docs/toolbox/accessibility_indicators/heatmaps/gravity.md

```diff
@@ -9,33 +9,34 @@ import thematicIcon from "/img/toolbox/data_management/join/toolbox.webp";
 
 # Heatmap - Gravity
 
-**Gravity-based Heatmaps** use a color-coded map to visualize the accessibility of [POIs](../../further_reading/glossary#point-of-interest-poi "What is a POI?") and point-based data.
+**Gravity-based Heatmaps** uses a color-coded map to visualize the accessibility of a points cluster, eg.: [POIs](../../further_reading/glossary#point-of-interest-poi "What is a POI?"). 
 
 ## 1. Explanation
 
-Visualized as a color-coded hexagonal grid, heatmaps take into account real-world transport and street networks to compute accessibility. When you specify a routing mode (such as `walking`, `bicycle`, etc.) and a maximum travel duration, a grid reflecting all areas accessible under these conditions is produced.
+Visualized as a color-coded hexagonal grid, heatmaps takes into account real-world transport and streets networks, to compute accessibility. Specify a routing mode (`Walk` or `Bicycle`), select `Opportunity Layer`. Set a maximum travel duration. The result shall display the color-coded hexagonal grid for all the areas accessible under those conditions. The color scale referes to the local accessibility.
+
+:::info INFO
```

Comment:
Minor, but a one-line gap needs to be left below the opening `:::info` and one line above the closing `:::`